src/RiotAPILibraries.ts

```diff
@@ -99,7 +99,7 @@ export default class RiotAPILibraries {
         const libraryResponse = await fetch(json.download_url);
         const libraryInfo: APILibraryStruct = await libraryResponse.json();
 
-        if (!libraryInfo.tags.some(tag => this.allTagOptions.includes(tag))) {
+        if (!libraryInfo.tags || libraryInfo.tags.some(tag => this.allTagOptions.includes(tag)) === -1) {
```

Comment:
This should be false, tsc is complaining about it.
`src/RiotAPILibraries.ts:102:34 - error TS2367: This condition will always return 'false' since the types 'boolean' and 'number' have no overlap.`

Reply:
oh good lord, I'm just not understanding what typescript wants me to do with these maybe-null/undefined (and those are apparently different types) values

Reply:
ok, TS was being more reasonable than I thought. yet another commit